docs/OPERATOR_GETTING_STARTED.md

```diff
@@ -60,9 +63,13 @@ ssh -i {key pair file name} ubuntu@{instance public IP}
 type yes when prompted
 ```
 
-### 3. Installation Script
+### 3. Pre-requisite
 
-Download and run the installation script.
+A fundamental pre-requisite is to have Docker installed on the machine hosting the SSV Node. In order to do so, please refer to [the official Docker documentation](https://docs.docker.com/engine/install/), and find the option that better fits your server configuration.
+
+> ℹ️ In order to run the SSV Node, in a server, only Docker engine is necessary, you can still go ahead and install Docker Desktop, but it will not be necessary unless you plan to use the Graphical Interface.
+
+As a shortcut, the installation script linked below can install the necessary packages for Ubuntu.
```

Comment:
thats my humble opinion, isn't just too much, feels like a lot of text 3 paragraphs to say -
 use the install script OR have docker installed.